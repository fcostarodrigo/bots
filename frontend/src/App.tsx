export default function App() {
  return <h1>Bots</h1>;
}
