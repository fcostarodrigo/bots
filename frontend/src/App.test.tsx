import { describe, expect, it } from "vitest";
import App from "./App";

describe(App, () => {
  it("should render", () => {
    expect(<App />).toMatchSnapshot();
  });
});
