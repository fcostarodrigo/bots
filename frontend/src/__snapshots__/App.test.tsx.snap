// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App > should render 1`] = `<App />`;
